nf_core/__main__.py

```diff
@@ -200,36 +201,33 @@ def launch(pipeline, id, revision, command_only, params_in, params_out, save_all
 
 
 # nf-core download
+
+
 @nf_core_cli.command(help_priority=3)
-@click.argument("pipeline", metavar="<pipeline name>")
+@click.argument("pipeline", required=False, metavar="<pipeline name>")
 @click.option("-r", "--release", type=str, help="Pipeline release")
 @click.option("-o", "--outdir", type=str, help="Output directory")
 @click.option(
-    "-c",
-    "--compress",
-    type=click.Choice(["tar.gz", "tar.bz2", "zip", "none"]),
-    default="tar.gz",
-    help="Archive compression type",
+    "-x", "--compress", type=click.Choice(["tar.gz", "tar.bz2", "zip", "none"]), help="Archive compression type"
 )
 @click.option("-f", "--force", is_flag=True, default=False, help="Overwrite existing files")
-@click.option("-s", "--singularity", is_flag=True, default=False, help="Download singularity images")
 @click.option(
-    "-c",
-    "--singularity-cache",
-    is_flag=True,
-    default=False,
-    help="Don't copy images to the output directory, don't set 'singularity.cacheDir' in workflow",
+    "-c", "--container", type=click.Choice(["none", "singularity"]), help="Download software container images"
+)
```

Comment:
Unless we are planning to extend this choice list at some point, this could also just be flag instead of a prompt I think.
But also fine leaving it as a prompt.

Reply:
My preference would be to leave it as a prompt. One could make a semi-reasonable case for offering `download` to pull e.g. docker images to the local instance.

Reply:
And yes I also wondered about docker too @pontus 👍🏻